src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceNamer.java

```diff
@@ -41,6 +42,15 @@ public JavaSurfaceNamer() {
   }
 
   @Override
+  public boolean shouldImportRequestObjectParamElementType(Field field) {
+    if (Field.IS_MAP.apply(field)) {
```

Comment:
Basically, I don't know. I'm thinking that if it turns out that PHP is the exception, we can move it up later.